docs/core/diagnostics/built-in-metrics-aspnetcore.md

```diff
@@ -11,45 +11,31 @@ This article describes the metrics built-in for ASP.NET Core produced using the
 <xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API. For a listing of metrics based on the older [EventCounters](event-counters.md) API,
 see [here](available-counters.md).
 
+ASP.NET Core metrics are grouped into meters:
```

Comment:
I like your suggestion to remove this link list from text knowing that the automatic formatting on the learn site creates very similar links.